test/runtests.jl

```diff
@@ -1277,6 +1277,32 @@ end
 
     @test sum(objective_weight(sp, disc) for sp in strat_periods(periods_unit)) ≈ 8.435 atol =
         1e-3
+
+    simple = SimpleTimes(10, 2)
+    for sp in strat_periods(simple)
+        for t in sp
+            @test discount(sp, simple, 0.05) == discount(t, simple, 0.05)
+            @test discount(sp, simple, 0.05; type = "avg") ==
+                  discount(t, simple, 0.05; type = "avg")
+        end
+    end
+
```

Comment:
```suggestion

    uniform_day = SimpleTimes(24, 1)
    periods = TwoLevel(3, 5, uniform_day, op_per_strat=8760)
```
To allow for a potential comparisons between tree and not tree.